linter/lintconsts.go

```diff
@@ -0,0 +1,49 @@
+package linter
+
+import (
+	"golang.org/x/crypto/ocsp"
+)
+
+type CertType string
+const (
+	Subscriber CertType = "Subscriber"
+	CA CertType = "CA"
+)
+
+type OCSPStatus string
+const (
+	Good OCSPStatus = "Good"
+	Revoked OCSPStatus = "Revoked"
+	None OCSPStatus = ""
+)
+
+type LintOpts struct {
+	LeafCertType CertType
+	LeafCertNonIssued bool
+	ExpectedStatus OCSPStatus
+}
+
+// LintStatus defines the possible statuses for a lint
+type LintStatus string
+
+const (
+	Passed LintStatus = "PASSED" // lint passed
+	Failed LintStatus = "FAILED" // lint failed
+	Unknown LintStatus = "UNKNOWN" // unknown whether lint passed or failed
+	Error  LintStatus = "ERROR"  // encountered error while running lint
+)
+
+// LintResult defines the struct of the result of a Lint
+type LintResult struct {
+	Lint   *LintStruct
+	Status LintStatus
+	Info   string
+}
+
+// StatusIntMap maps ocsp statuses to strings
+var StatusIntMap = map[int]string{
+	ocsp.Good:    "good",
+	ocsp.Revoked: "revoked",
+	ocsp.Unknown: "unknown",
```

Comment:
Why are you setting constants here for a different file?  Shouldn't everywhere that was referencing the constants from ocsp,  now reference them from here?

Also weren't these capitalized in another PR. (e.g. `"good"` -> `"Good"`)

Reply:
Actually they were converted to lowercase in https://github.com/googleinterns/ocsp-response-linter/pull/21/files. So I guess they are consistent. These values seem very similar to lines 14-17. Is there a reason they appear separate?